js-core-1/week-2/lesson.md

```diff
@@ -237,262 +294,210 @@ function satisfiesRequirements(num) {
 }
 ```
 
-We can test expressions with logical operators in a node console too:
+We can test expressions with logical operators in a node console too.
 
-```sh
-$ node
-> var num = 10;
-undefined
-> num > 5 && num < 15
-true
-> num < 10 || num === 10
-true
-> false || true
-true
-> !true
-false
-> var greaterThan5 = num > 5
-undefined
-> !greaterThan5
-false
-> !(num === 10)
-false
-```
+### Exercise (5 mins)
+Type the following expressions into your node REPL and note the output. Anything you didn't expect?
+* `let num = 10`
+* `num > 5 && num < 15`
+* `num < 10 || num === 10`
+* `false || true`
+* `!true`
+* `let greaterThan5 = num > 5`
+* `!greaterThan5`
+* `!(num === 10)`
+
+### Exercise (15 mins)
+In pairs, write a function that checks a username is of an acceptable format for a user type. The function must:
+
+* take two parameters: one for the username and one for the user type
+* if the username starts with a capital letter _and_ has length between 5 and 10 characters long, it must return `"Username valid"`; otherwise, it must return `"Username invalid"`
+* **BONUS:** if the user type is an `admin` or a `manager`, all usernames must return `"Username valid"`
 
-## More Conditionals
 
-A common use of if statements is inside of functions.
+## Loops
+
+When we're writing programs, we often find that we want to repeat a bit of code over and over, or repeat it but change something about it each time. To save ourselves from writing all that code, we can use a **loop**. JavaScript has two kinds of loops, a `while` loop and a `for` loop.
+
+A `while` loop is a way to repeat code until some condition is evaluated to `false`. For example:
 
 ```js
-function getGrade(score) {
-  if (score >= 80) {
-    return "A";
-  }
-  if (score >= 60) {
-    return "B";
-  }
+let i = 0;
+while (i < 20) {
+  someFunction(i);
+  i++;
 }
 ```
 
-You can also write this using `else if`:
+It's important that the condition inside the parenthesis becomes false at some point - otherwise we'll have what's known as an infinite loop!
```

Comment:
```suggestion
It's important that the condition inside the `while` condition becomes false at some point - otherwise we'll have what's known as an infinite loop!
```

Reply:
> condition inside the `while` condition

Sounds redundant?